pkg/util/encryption/multi_test.go

```diff
@@ -0,0 +1,91 @@
+package encryption
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	mock_keyvault "github.com/Azure/ARO-RP/pkg/util/mocks/keyvault"
+)
+
+func TestNewMulti(t *testing.T) {
```

Comment:
I'm not sure that it is benefitial to test `NewMulti` as it consists of calls of functions tested elsewhere (`NewAES256SHA512` & `NewXChaCha20Poly1305` for example).

We should, however test that we use multiple openers (mocked openers should be enough, I think - no need to instantiate real ones).

Reply:
I agree that this one is a little redundant, but maybe that's not a bad thing. I'll add a test for `(c *multi) Open`.